templates/firefox-ios.xliff

```diff
@@ -3061,22 +3117,174 @@
+        <source>Recommended by Pocket</source>
+        <note>Title for Firefox Pocket stories widget in Gallery View where user can add it to home screen. Pocket is the name of another app.</note>
+      </trans-unit>
       <trans-unit id="TodayWidget.PrivateTabButtonLabelV1">
         <source>Private Search</source>
         <note>Open New Private Tab button label</note>
       </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionDescription">
+        <source>Select a Firefox shortcut to add to your Home screen.</source>
+        <note>Quick action description when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionDropDownMenu">
+        <source>Quick action</source>
+        <note>Quick Actions left label text for dropdown menu when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionGalleryDescription">
+        <source>Add a Firefox shortcut to your Home screen. After adding the widget, touch and hold to edit it and select a different shortcut.</source>
+        <note>Description for small size widget to add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionsGalleryTitle">
+        <source>Quick Actions</source>
+        <note>Quick Actions title when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionsSubLabel">
+        <source>Firefox - Quick Actions</source>
+        <note>Sub label for medium size quick action widget</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewGalleryDescription">
+        <source>Access your open tabs directly on your homescreen.</source>
+        <note>Description for Quick View widget in Gallery View where user can add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewGalleryTitle">
+        <source>Quick View</source>
+        <note>Title for Quick View widget in Gallery View where user can add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewLargeGalleryDescription">
+        <source>Add shortcuts to your open tabs.</source>
+        <note>Description for Quick View widget in Gallery View where user can add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickViewOpenTabsSubLabel">
+        <source>Firefox - Open Tabs</source>
+        <note>Sub label for Top Sites widget</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.SearchInFirefoxV2">
+        <source>Search in
+Firefox</source>
+        <note>Search in Firefox. Do not translate the word Firefox</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.SearchInPrivateTabLabelV2">
+        <source>Search in
+Private Tab</source>
+        <note>Search in private tab</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.SearchInTitle">
+        <source>Search in</source>
+        <note>Search</note>
```

Comment:
Please add note saying what the entire string will be (so, what's missing in it), so localizers have context on what's missing here.

Reply:
Addressing this in https://github.com/mozilla-l10n/firefoxios-l10n/compare/master...nbhasin2:20200819_1332?expand=1